components/layout_2020/positioned.rs

```diff
@@ -928,9 +928,13 @@ impl<'a> AbsoluteAxisSolver<'a> {
             _ => return None,
         };
 
+        let default_overflow_rect = (alignment_container.origin.max(Au::zero())) +
+            (alignment_container.length.max(Au::zero()));
```

Comment:
Disregard the last part, for static position it's not necessarily `auto`.
I think then it would be

```rs
self.containing_size.max(static_position_rect_axis.origin + static_position_rect_axis.length)
    - static_position_rect_axis.origin.min(Au::zero())
```

A helper method for `RectAxis` that returns the end position could be useful.